test/variables.tf

```diff
@@ -11,7 +11,7 @@ variable "envid" {
 }
 
 variable "bootstrap_version" {
-  default = "master"
+  default = "bump-dataog-agent"
```

Comment:
this require merge of: https://github.com/aeternity/infrastructure/pull/555 then it will be set back to master